Assets/ZSSRichTextEditor.js

```diff
@@ -274,8 +272,12 @@ ZSSEditor.getSelectedText = function() {
 ZSSEditor.getCaretArguments = function() {
     var caretInfo = this.getYCaretInfo();
     
-    this.caretArguments[0] = 'yOffset=' + caretInfo.y;
-    this.caretArguments[1] = 'height=' + caretInfo.height;
+    if (caretInfo == null) {
+        this.caretArguments = null;
```

Comment:
It looks like `caretArguments` starts off as an array with 0 for the yOffset and height, but if caretInfo is ever null then caretArguments is also set to null.  Maybe it should just start out as null?